### PR TITLE
HIVE-23922: Improve code quality, UDFArgumentException.getMessage Method requires only two parameters

### PR DESCRIPTION
**Pull request by dh20**
## NOTICE

Please create an issue in ASF JIRA before opening a pull request,
and you need to set the title of the pull request which starts with
the corresponding JIRA issue number. (e.g. HIVE-XXXXX: Fix a typo in YYY)
For more details, please see https://cwiki.apache.org/confluence/display/Hive/HowToContribute

### CODE REVIEW

**Comment by dh20** [comment]
Someone help me to have a look, this error and I submit seems to have nothing to do with it! very thank!

**Comment by pvary** [comment]
> Someone help me to have a look, this error and I submit seems to have nothing to do with it! very thank!

Just retrigger the test.

**Comment by dh20** [comment]
@pvary These errors are not caused by my changes. Can you help me to have a look? Thank you very much

**Comment by pvary** [comment]
@dh20 Green run is needed for pushing the changes. Please retrigger :(

**Comment by dh20** [comment]
@pvary The test has passed

**Comment by dh20** [comment]
Hi, @sunchao 
Could you review this PR, please?

**Comment by sunchao** [comment]
@dh20 can you fix the title and the JIRA number? HIVE-23896 seems unrelated to this PR.

**Comment by dh20** [comment]
> @dh20 can you fix the title and the JIRA number? [HIVE-23896](https://issues.apache.org/jira/browse/HIVE-23896) seems unrelated to this PR.

@sunchao Sorry, I made a mistake in Jire number. Now I have corrected it

**Comment by sunchao** [comment]
@dh20 looks good - can you remove `typeNames` as well in the method body? it is also not used.

**Comment by dh20** [comment]
> can you remove `typeNames` as well in the method body? it is also not used.


> @dh20 looks good - can you remove `typeNames` as well in the method body? it is also not used.

Yes, my colleagues have submitted this question to pr [HIVE-23996]

**Comment by dh20** [comment]
@sunchao thanks very mach!

**Comment by dh20** [comment]
@sunchao The test has passed

File: udf/src/java/org/apache/hadoop/hive/ql/exec/UDFArgumentException.java [hunk 1]
```diff
@@ -65,16 +65,13 @@ public UDFArgumentException(String message,
...
     this.funcClass = funcClass;
     this.argTypeInfos = argTypeInfos;
     this.methods = methods;
   }
-  
-  private static String getMessage(String message, 
-      Class<?> funcClass,
-      List<TypeInfo> argTypeInfos,
-      List<Method> methods) {
+  //HIVE-23896 remove unnecessary parameter
```
Comment (pgaref) [inline udf/src/java/org/apache/hadoop/hive/ql/exec/UDFArgumentException.java:73]: Could you please remove the comment and replace with a new line?

**Pull request merged**